prow/config/jobs/istio.yaml

```diff
@@ -1,179 +1,202 @@
 org: istio
 repo: istio
-image: gcr.io/istio-testing/build-tools:2019-10-11T13-37-52
-branches:
-  - master
+support_release_branching: true
 
 jobs:
   - name: unit-tests
-    command: [make, -e, "T=-v", build, localTestEnv, racetest, binaries-test]
+    command: [entrypoint, make, -e, "T=-v", build, localTestEnv, racetest, binaries-test]
```

Comment:
rather than `entrypoint` we could make this a new `requirement: [docker]` to be more explicit *why* we have it